### PR TITLE
KubeCon 2017: glossary entry for Persistent Volume Claim

### PR DESCRIPTION
**Pull request by erwbgy**
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> For 1.9 Features: set Milestone to `1.9` and Base Branch to `release-1.9`
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> NOTE: Please check the “Allow edits from maintainers” box (see image below) to
> [allow reviewers to fix problems](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) on your patch and speed up the review process.
>
> Please delete this note before submitting the pull request.
>
> NOTE: After opening the PR, please *un-check and re-check* the "Allow edits from maintainers" box. This is a temporary workaround to address a known issue with GitHub.

Allow edits from maintainers checkbox

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/6598)

### CODE REVIEW

File: _data/glossary/persistent-volume-claim.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+related:
+- persistent-volume
+- statefulset
+- deployment
+- pod
+tags:
+- core-object
+- storage
+short-description: >
+  A PersistentVolumeClaim (PVC) is used to claim storage resources defined in a PersistentVolume (PV) so that it can be mounted as a volume in a container.
```
Comment (tonglil) [inline _data/glossary/persistent-volume-claim.yaml]: Drop "A PersistentVolumeClaim (PVC) is" to avoid repeating the term.

Comment (tonglil) [inline _data/glossary/persistent-volume-claim.yaml]: Link to the glossary item for `PV` once https://github.com/kubernetes/website/pull/6591 is merged:

`{% glossary_tooltip text="Persistent Volume" term_id="persistent-volume" %}`

File: _data/glossary/persistent-volume-claim.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+long-description: |
+  The PersistentVolumeClaim (PVC) specifies the amount of storage, how the storage will be accessed (read-only, read-write and/or exclusive) and how it is reclaimed (retained, recycled or deleted), but no details of the storage itself (which are in the PV).
```
Comment (tonglil) [inline _data/glossary/persistent-volume-claim.yaml]: `(which are in the PV)` -> `, which is configured by a Persistent Volume.`

**Comment by tonglil** [comment]
Netlify failed but it seems to be transient issue unrelated to the changes.
Push again to rerun Netlify.

File: _data/glossary/persistent-volume-claim.yaml (hunk 2 above)
Comment (Bradamant3) [reply _data/glossary/persistent-volume-claim.yaml]: Better yet, start another sentence at this point. End the first sentence where you currently have a comma (after `deleted)`. Then add something like `You provide the details of the storage in the Persistent Volume specification.`

File: _data/glossary/persistent-volume-claim.yaml (hunk 1 above)
Comment (Bradamant3) [reply _data/glossary/persistent-volume-claim.yaml]: And to clarify, start your (new) fragment with "Claims the storage resources that are defined by a PersistentVolume object ..." (no need to include "is used," which adds no meaning)

**Comment by abiogenesis-now** [comment]
Thank you for the change @erwbgy ! 

You might be wondering why the Netlify preview is breaking, so here's some context: the `glossary_tooltip` tag sets up a bit of a merge dependency, since the other glossary term you are referencing *has* to already exist. In other words, your current PR requires the merge of https://github.com/kubernetes/website/pull/6591 to build properly.

You can either:
(A) Wait for said PR to get revised and merged
(B) Remove the `glossary_tooltip` from this PR (which SIG docs maintainers will add back in after-the-fact), so that this can be merged right away.

We're fine with either approach. Let us know if you have any questions, and apologies for the confusion!

**Pull request merged**